driver/src/bin/stablex.rs

```diff
@@ -10,7 +10,7 @@ use std::time::Duration;
 fn main() {
     simple_logger::init_with_level(log::Level::Info).unwrap();
 
-    let contract = StableXContractImpl::new().unwrap();
+    let (contract, _event_loop) = BatchExchange::new().unwrap();
```

Comment:
I will be excited to see what this means if we wind up doing it!

Reply:
Hmm, my problem with that is that if we ever end up wanting to create and clean up multiple contract instances, then it might be problematic as they would never get cleaned up. I suggest we do `event_loop.into_remote()` for now, but then we have to keep in mind that these event loops are leaking every time we make a new contract instance.